qemu/tests/cfg/timedrift_check_clock_offset.cfg

```diff
@@ -34,3 +34,25 @@
                     nmi_cmd = "guest: echo '1' > /proc/sys/kernel/sysrq && echo '0' > /proc/sys/kernel/panic && service kdump stop; echo 'c' > /proc/sysrq-trigger"
                     ntp_cmd = "ntpdate ${ntp_server}"
                     ntp_query_cmd = "ntpdate -q ${ntp_server}"
+        - hotplug_vcpu:
+            type = timedrift_check_when_hotplug_vcpu
+            start_vm = yes
+            ntp_server = "clock.redhat.com"
+            vcpu_sockets = 2
+            smp = 4
```

Comment:
To aviod vcpu overcommit for single guest.  should set vcpu_sockets =1 and smp=1.  In addition,  ntpdate package isn't available for RHEL8. not sure which tool can connect ntp server.